plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -217,22 +211,23 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 		D3D_FEATURE_LEVEL_10_0,
 	};
 
-	hr = D3D11CreateDeviceAndSwapChain(adapter, driver_type, NULL, flags, levels, countof(levels), D3D11_SDK_VERSION, &scd, &m_swapchain, &m_dev, &level, &m_ctx);
+	hr = D3D11CreateDeviceAndSwapChain(adapter, driver_type, NULL, flags, levels, countof(levels), D3D11_SDK_VERSION, &swap_chain_desc, &m_swapchain, &m_dev, &level, &m_ctx);
 
-	if(FAILED(hr)) return false;
+	if (FAILED(hr))
+		return false;
```

Comment:
It would be nice to have a sort of macro that expands to something like this code.
And additionally some logging in it would be nice
ie log the function name and the result code